libraries/AP_Logger/AP_Logger.h

```diff
@@ -498,6 +501,8 @@ class AP_Logger
     bool fill_log_write_logstructure(struct LogStructure &logstruct, const uint8_t msg_type) const;
 
     bool _armed;
+    bool _motor_test_active;
+    uint32_t _last_motor_test_end;
```

Comment:
```suggestion
    uint32_t _last_motor_test_end_ms;
```
We should probably add `ms` to give the unit.